docs/components/empty-state.md

```diff
@@ -115,6 +114,52 @@ export default {
 </script>
 ```
 
+### is-error
+- `is-error`
+
+A flag denoting whether or not the message is an error message. If so, a warning icon is displayed above the title slot.
+
+<template>
+  <KEmptyState is-error="true">
+    <h5 slot="message">
+      Error: Something broke
+    </h5>
+  </KEmptyState>
+</template>
+
+```vue
+<template>
+  <KEmptyState is-error="true">
+    <h5 slot="message">
+      Error: Something broke
+    </h5>
+  </KEmptyState>
+</template>
+```
+
+### icon-size
+- `icon-size`
+
+A number denoting the size of the warning icon to be displayed above the error message. The default size is 20.
```

Comment:
default icon size should be bigger looking at the current KM error state:
![image](https://user-images.githubusercontent.com/5770711/75460418-70530980-594f-11ea-8254-b0f9a58fb1d5.png)

Reply:
keep in mind we are using KIcon's warning which is a little different than our previous warning icon